src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -206,15 +206,19 @@ def kubernetes_exception_handler(ex, fault_type, summary, error_message='Error o
         status_code = ex.status
         if status_code == 403:
             logger.warning(message_for_unauthorized_request)
-        if status_code == 404:
+        elif status_code == 404:
             logger.warning(message_for_not_found)
+        else:
+            logger.debug("Kubernetes Exception: " + str(ex))
         if raise_error:
             telemetry.set_exception(exception=ex, fault_type=fault_type, summary=summary)
             raise CLIInternalError(error_message + "\nError Response: " + str(ex.body))
```

Comment:
Shouldn't we choose an error type from 'UserFault' category since its a case of user fault?

Reply:
Do we want to error out here? That branching is not part of raise_error.

Reply:
This I did just to fix the remaining situation cases, out to debug log.